ds701_book/17-Regression-I-Linear.qmd

```diff
@@ -140,67 +287,242 @@ plt.text(-9, 3, r'$y = \beta_0 + \beta_1x$', size=20);
+# Title
+#ax.set_title('Linear Regression with Residuals')
+
+# Add legend
+#ax.legend()
+
+# Show the plot
+plt.show()
+```
+
+
+:::
+::: {.column width="50%"}
 
-Image from Lay, _Linear Algebra and its Applications,_ 4th edition
 
 We call 
+
 * $y_j$ the __observed__ value of $y$ and 
 * $\beta_0 + \beta_1 x_j$ the __predicted__ $y$-value.   
 
-The difference between an observed $y$-value and a predicted $y$-value is called a __residual__.
+:::
+::::
+
+:::: {.fragment}
+The difference between an observed $y$-value and a predicted $y$-value is called
+a __residual__.
+::::
 
+:::: {.fragment}
 There are several ways of measure how "close" the line is to the data. 
+::::
 
+:::: {.fragment}
 The usual choice is to sum the squares of the residuals.  
+::::
 
-The __least-squares line__ is the line $y = \beta_0 + \beta_1x$ that minimizes the sum of squares of the residuals. 
+:::: {.fragment}
+The __least-squares line__ is the line $y = \beta_0 + \beta_1x$ that minimizes
+the sum of squares of the residuals. 
 
-The coefficients $\beta_0, \beta_1$ of the line are called __regression coefficients.__
+The coefficients $\beta_0, \beta_1$ of the line are called
+__regression coefficients.__
+::::
 
-__A least-squares problem.__
+## A least-squares problem
 
 If the data points were on the line, the parameters $\beta_0$ and $\beta_1$ would satisfy the equations
 
-$$\beta_0 + \beta_1 x_1 = y_1 $$
-$$\beta_0 + \beta_1 x_2 = y_2 $$
-$$\beta_0 + \beta_1 x_3 = y_3 $$
-$$ \vdots$$
-$$\beta_0 + \beta_1 x_n = y_n $$
+$$
+\begin{align*}
+\beta_0 + \beta_1 x_1 &= y_1 \\
+\beta_0 + \beta_1 x_2 &= y_2 \\
+\beta_0 + \beta_1 x_3 &= y_3 \\
+&\vdots \\
+\beta_0 + \beta_1 x_n &= y_n
+\end{align*}
+$$
 
 We can write this system as 
 
-$$X\mathbf{\beta} = \mathbf{y}$$
+$$
+X\mathbf{\beta} = \mathbf{y}
+$$
 
 where 
 
-$$X=\begin{bmatrix}1&x_1\\1&x_2\\\vdots&\vdots\\1&x_n\end{bmatrix},\;\;\mathbf{\beta} = \begin{bmatrix}\beta_0\\\beta_1\end{bmatrix},\;\;\mathbf{y}=\begin{bmatrix}y_1\\y_2\\\vdots\\y_n\end{bmatrix}$$
+$$
+X=
+\begin{bmatrix}
+1 & x_1 \\
+1 & x_2 \\
+\vdots & \vdots \\
+1 & x_n
+\end{bmatrix},
+\;\;\mathbf{\beta} = \begin{bmatrix}\beta_0\\\beta_1\end{bmatrix},
+\;\;\mathbf{y}=\begin{bmatrix}y_1\\y_2\\\vdots\\y_n\end{bmatrix}
+$$
+
+---
 
 Of course, if the data points don't actually lie exactly on a line, 
 
-... then there are no parameters $\beta_0, \beta_1$ for which the predicted $y$-values in $X\mathbf{\beta}$ equal the observed $y$-values in $\mathbf{y}$, 
+... then there are no parameters $\beta_0, \beta_1$ for which the predicted
+$y$-values in $X\mathbf{\beta}$ equal the observed $y$-values in $\mathbf{y}$, 
 
 ... and $X\mathbf{\beta}=\mathbf{y}$ has no solution.
 
-Now, since the data doesn't fall exactly on a line, we have decided to seek the $\beta$ that minimizes the sum of squared residuals, ie,
+---
+
+Now, since the data doesn't fall exactly on a line, we have decided to seek the
+$\beta$ that minimizes the sum of squared residuals, ie,
```

Comment:
i.e., rather than ie,

I've been conditioned to write this way from my thesis and paper writing.